app/src/main/java/com/framgia/soundcloud/screen/listtracks/ListtracksContract.java

```diff
@@ -27,5 +26,7 @@
         void getTracks();
 
         void onDestroy();
+
+        void LoadMoreTracks(String nextHref);
```

Comment:
ERROR: [MethodNamingConventions] Method names should not start with capital letters
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#MethodNamingConventions